README.md

```diff
@@ -78,37 +78,64 @@ kurtosis enclave stop lambdanet ; kurtosis enclave rm lambdanet
 ## Roadmap
 
 ### Milestone 1: RPC Node
-Add support to participate in a Cancun localnet as a read-only node.
+Add support to follow a post-Merge localnet as a read-only RPC Node. This first milestone will only support a canonical chain (every incoming block has to be the child of the current head). 
 
 RPC endpoints
-- `engine_newPayloadV3` (partially)
+- `engine_newPayloadV3` (excl. block building)
+- `eth_blobBaseFee`
+- `eth_blockNumber`
+- `eth_call`
+- `eth_chainId`
+- `eth_createAccessList`
+- `eth_estimateGas`
+- `eth_feeHistory`
+- `eth_getBalance` (at head block)
 - `eth_getBlockByHash`
 - `eth_getBlockByNumber`
-- `eth_blockNumber`
```

Comment:
why remove this? It is currently supported

Reply:
I just sorted them alphabetically